README.md

```diff
@@ -102,18 +102,26 @@ npm install
 
 ## Faster development building with ic-design-system
 
-If you want to build the components in this repository and use them in [`ic-design-system`](https://github.com/mi6/ic-design-system), you can use `npm link`. This will speed up development because you do not have to generate a compressed npm package, but rather directly use the package after building it locally.
+If you want to build the components in this repository and use them in [`ic-design-system`](https://github.com/mi6/ic-design-system), you can use `npm link`.
+This will speed up development because you do not have to generate a compressed npm package, but rather directly use the package after building it locally.
+You will need to rebuild the package after every change though.
```

Comment:
this caught me out as well, I expected `link` to mean I could just refresh my page on change but it doesn't work.
The previous sentence implies that it won't work but this is more explicit.

Reply:
True, but it does say that you don't need the npm pack result. I added this and thought running `npm run build:all` was self-explainatory, but nice change! 👍🏾